### PR TITLE
APP-403: [Java] Credentials demos fix bugs in apps and documentation

### PR DESCRIPTION
**Pull request by Rib47**
…Remove needless dependencies.

### CODE REVIEW

**Review by makswinner — APPROVED** [review]

File: credentialsdemo/assembly/src-java.xml [hunk 1]
```diff
@@ -36,17 +36,4 @@
             <outputDirectory>JCredentialsDemo/lib</outputDirectory>
         </fileSet>
     </fileSets>
-    <dependencySets>
```
Comment (Slash32) [inline credentialsdemo/assembly/src-java.xml:39]: Please identify why is this dropped?

**Review by Slash32 — CHANGES_REQUESTED** [review]
Small question, otherwise good.

**Comment by Slash32** [comment]
@kaa-leeroy test this fully please

File: credentialsdemo/assembly/src-java.xml (hunk 1 above)
Comment (Rib47) [reply credentialsdemo/assembly/src-java.xml:39]: This section copies dependencies that needed only for Admin application.
Client application do not need them.

Comment (Slash32) [reply credentialsdemo/assembly/src-java.xml:39]: Thanks for the explanation

**Review by Slash32 — APPROVED** [review]

**Pull request merged**